VAMobile/src/store/slices/authSlice.ts

```diff
@@ -694,17 +635,10 @@ export const handleTokenCallbackUrl =
           grant_type: 'authorization_code',
           code_verifier: getState().auth.codeVerifier ?? '',
           code,
-          ...(!SISEnabled
-            ? {
-                client_id: AUTH_IAM_CLIENT_ID,
-                client_secret: AUTH_IAM_CLIENT_SECRET,
-                redirect_uri: AUTH_IAM_REDIRECT_URL,
-              }
-            : {}),
         }).toString(),
       })
       const authCredentials = await processAuthResponse(response)
-      await logAnalyticsEvent(Events.vama_login_success(SISEnabled))
+      await logAnalyticsEvent(Events.vama_login_success(true))
```

Comment:
at the moment we do not, but I didn't want to drop that as I wasn't sure if any login changes were going to happen. This might be a good thing to address when we redo login, but should be find to keep for this ticket.